src/bindgen/parser.rs

```diff
@@ -524,21 +526,25 @@ impl Parse {
                     self.load_syn_ty(crate_name, mod_cfg, item);
                 }
                 syn::Item::Impl(ref item_impl) => {
-                    for item in &item_impl.items {
-                        if let syn::ImplItem::Const(ref item) = item {
-                            self.load_syn_assoc_const(
-                                binding_crate_name,
-                                crate_name,
-                                mod_cfg,
-                                &item_impl.self_ty,
-                                item,
-                            );
-                        }
-                    }
+                    impls.push(item_impl);
```

Comment:
Can we minimize our memory usage here by filtering out `Impl`s that don't contain associated constants?

I'm not too worried about it though, as IIRC `rustc` needs to hold the full AST in memory, so if that's too much for us it should be too much for them as well.

Reply:
I think this should be handled now.